qiita_db/metadata_template/test/test_prep_template.py

```diff
@@ -735,6 +735,65 @@ def test_clean_validate_template(self):
         exp.sort_index(axis=1, inplace=True)
         assert_frame_equal(obs, exp)
 
+    def test_clean_validate_template_no_forbidden_words1(self):
+        PT = qdb.metadata_template.prep_template.PrepTemplate
+        self.metadata.rename(columns={'center_name': 'sampleid'}, inplace=True)
+        with self.assertRaises(qdb.exceptions.QiitaDBColumnError):
+            PT._clean_validate_template(self.metadata, 2)
+
+    '''
+    Commenting these two out for now. There is legacy code in
```

Comment:
(*) see comments above. BTW I'm OK with removing/replacing these tests as long as we have some to make sure that the automatic drop happens ...

Reply:
Depending on whether it's better to move the silent drop a level above the _clean_validate_template, I'd say yes, we could drop these tests, and write ones to test the automatic drop instead.

Reply:
Omitting automatic drop test for now; may need to shuffle tests around in clean_validate_template first. That dovetails with another issue I found earlier that I want to make an issue in GitHub.